### ISSUE TITLE
Stuck at Running Repository Initialize

### ISSUE THREAD

**Issue by neo** [github-for-unity/Unity#150]
Unity 2017.1.0f3 Personal (64bit) macOS
```
170728-17:06:34  INFO [ 1] <Global>                            Initializing GitHub for Unity version 0.15.0.0
170728-17:06:34 TRACE [ 1] <IApplicationManager>               Run - CurrentDirectory /Users/admin/Desktop/unity/Tanks copy 2
170728-17:06:34 TRACE [ 1] <GitInstaller>                      SetupIfNeeded
170728-17:06:34 TRACE [ 1] <GitInstaller>                      SetupGitIfNeeded
170728-17:06:34 TRACE [ 1] <GitInstaller>                      /Users/admin/.local/share/GitHubUnity/PortableGit_f02737a78695063deace08e96d5042710d3e32db/bin/git not installed yet
170728-17:06:34  WARN [ 1] <GitInstaller>                      Archive "Assets/Editor/GitHub/PlatformResources/mac/git.zip" missing
170728-17:06:34  WARN [ 1] <GitInstaller>                      Archive "Assets/Editor/GitHub/Assets/Editor/GitHub/PlatformResources/mac/git.zip" missing, returning
170728-17:06:34 TRACE [ 1] <GitInstaller>                      SetupGitLfsIfNeeded
170728-17:06:34 TRACE [ 1] <GitInstaller>                      GitLFS MD5: 54F586DF05A4B80FEAE4FABED10D8E95
170728-17:06:34 TRACE [ 1] <GitInstaller>                      GitLFS Present
170728-17:06:34 TRACE [ 1] <GitInstaller>                      Already extracted /Users/admin/.local/share/GitHubUnity/PortableGit_f02737a78695063deace08e96d5042710d3e32db/libexec/git-core/git-lfs, returning
170728-17:06:34 TRACE [ 1] <GitClient>                         Git Installation folder  discovered: '/usr/local/bin/git'
170728-17:06:34 TRACE [ 1] <IApplicationManager>               Environment.GitExecutablePath "/Users/admin/.local/share/GitHubUnity/PortableGit_f02737a78695063deace08e96d5042710d3e32db/bin/git" Exists:False
170728-17:06:34 TRACE [ 1] <IApplicationManager>               Restarted 
170728-17:06:34 TRACE [ 1] <Window>                            Initialize ApplicationManager:GitHub.Unity.ApplicationManager Initialized:True
170728-17:06:55 TRACE [ 1] <IApplicationManager>               Running Repository Initialize
```

**Comment by StanleyGoldman** [comment]
Hey @neo, thanks for the bug report. I don't know when we will be able to add support for 2017.1, but it will hopefully be soon.

**Comment by StanleyGoldman** [comment]
Fixed with #175
